sql/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -1253,6 +1258,21 @@ public void testAlterTableAddColumnWithCheckConstraint() throws Exception {
                      Maps.getByPath(mapping, Arrays.asList("_meta", "check_constraints", "bazinga_check")));
     }
 
+    @Test
+    public void testAlterTableDropCheckConstraint() throws Exception {
+        SQLExecutor.builder(clusterService)
+            .addTable("create table t (" +
+                      "    id int primary key, " +
+                      "    qty int constraint check_qty_gt_zero check(qty > 0), " +
+                      "    constraint check_id_ge_zero check (id >= 0)" +
+                      ")")
+            .build();
+        analyze("alter table t drop constraint check_qty_gt_zero");
+        expectedException.expectMessage(
+            "Cannot find a CHECK CONSTRAINT named [check_id_ge_zero_what], available names are: [check_id_ge_zero, check_qty_gt_zero]");
```

Comment:
Why does this still list `check_qty_gt_zero` in available names?

Reply:
Because the constraint is eliminated at a lower level, here we are only testing the analyzer, which does not affect the table definition.
I will add this select also to the DDLIntegrationTest, where the error message should contain one less constraint indeed.

Reply:
In fact I already to in DDLIntegrationTest.test_alter_table_drop_constraint_removes_the_constraint_and_leaves_other_constraints_in_place